var/spack/repos/builtin/packages/r-tidyr/package.py

```diff
@@ -31,14 +31,18 @@ class RTidyr(RPackage):
     pipelines."""
 
     homepage = "https://github.com/hadley/tidyr"
-    url      = "https://cran.r-project.org/src/contrib/tidyr_0.5.1.tar.gz"
+    url      = "https://cran.r-project.org/src/contrib/tidyr_0.7.2.tar.gz"
     list_url = "https://cran.r-project.org/src/contrib/Archive/tidyr"
 
+    version('0.7.2', '42d723bf04c5c1c59e27a8be14f3a6b6')
     version('0.5.1', '3cadc869510c054ed93d374ab44120bd')
 
     depends_on('r-tibble', type=('build', 'run'))
     depends_on('r-dplyr', type=('build', 'run'))
     depends_on('r-stringi', type=('build', 'run'))
     depends_on('r-lazyeval', type=('build', 'run'))
     depends_on('r-magrittr', type=('build', 'run'))
+    depends_on('r-glue', type=('build', 'run'))
     depends_on('r-rcpp', type=('build', 'run'))
+    depends_on('r-purrr', type=('build', 'run'))
+    depends_on('r-tidyselect', type=('build', 'run'))
```

Comment:
Missing a dependency on `r-rlang`.

Reply:
r-rlang is added to dependency now. @adamjstewart